code/game/objects/random/voidsuit.dm

```diff
@@ -8,7 +8,7 @@
 
 /obj/random/voidsuit/item_to_spawn()
 	return pickweight(list(/obj/item/clothing/suit/space/void = 2,
-		/obj/item/clothing/suit/space/void/engineering = 1,
```

Comment:
Why this?

Reply:
People told me to make it more common because it's no longer really, really good compared to most other voids, and I agreed, though I can set it back.